app/src/main/java/org/xwiki/android/sync/activities/OIDC/OIDCActivity.kt

```diff
@@ -214,8 +223,21 @@ class OIDCActivity: AppCompatActivity(), AccountClickListener {
             selectedAccountName = accountName
             val flow: AuthorizationCodeFlow = createAuthorizationCodeFlow(accountName, serverUrl)
             flow.newAuthorizationUrl().setRedirectUri(REDIRECT_URI).build().let {
-                val browserIntent = Intent(Intent.ACTION_VIEW, Uri.parse(it))
-                startActivity(browserIntent)
+                runOnUiThread {
+                    binding.webview.webViewClient = object : WebViewClient() {
```

Comment:
Will be better to extract this to outside class

Reply:
Done